code/datums/diseases/Cancer

```diff
@@ -0,0 +1,45 @@
+/datum/disease/Cancer
+	form = "Condition"
+	name = "Cancer"
+	max_stages = 5
+	cure_text = "Surgery"
+	agent = "Cancer Cells"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 1
+	desc = "If left untreated the subject will become very weak,Massive Organ Failure and Die."
```

Comment:
grammar in this sentence.
If left untreated the subject will become very weak, have an organ failure and die.
Something the like